### PR TITLE
[QACI-5504] - 5.30.0 Release Notes + TCK Results

### PR DESCRIPTION
**Pull request by cubastanley**
Title

### CODE REVIEW

File: docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+= Release notes - Payara Platform Development - Version 5.30.0
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc:1]: Please change to:

Release Notes - Payara Platform Enterprise 5.30.0

File: docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * MicroProfile 4.1
+
+### Bug Fixes
+* [FISH-5654] - NullPointerException when setting server log format to JSON
+* [FISH-1530] - Fix OpenAPI TCK Tag Collection Test
+* [FISH-1521] - Fix Incorrect Group/Role Mapping in OIDC Provider
+* [FISH-1520] - Fix MicroProfile JWT-Auth TCK Failures
+* [FISH-1312] - MP OpenAPI schema property hidden is ignored
+* [FISH-1204] - Missing example values in OpenApi components schema definition using nested objects
+* [FISH-1068] - Parameter Serialization error in open api document for query parameter
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc:16]: I think this should be:

OpenAPI  (all one word and with capitals)

File: docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ * Jakarta EE 8
+ * MicroProfile 4.1
+
+### Bug Fixes
+* [FISH-5654] - NullPointerException when setting server log format to JSON
+* [FISH-1530] - Fix OpenAPI TCK Tag Collection Test
+* [FISH-1521] - Fix Incorrect Group/Role Mapping in OIDC Provider
+* [FISH-1520] - Fix MicroProfile JWT-Auth TCK Failures
+* [FISH-1312] - MP OpenAPI schema property hidden is ignored
+* [FISH-1204] - Missing example values in OpenApi components schema definition using nested objects
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc:15]: I think it should be:
OpenAPI
(all one word and with capitals)

**Review by debbiehoffman — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* [FISH-1068] - Parameter Serialization error in open api document for query parameter
+* [FISH-1066] - ConcurrentModificationException is thrown for config key "org.eclipse.microprofile.config.ConfigValue:mp.config.profile:60s" on deploying REST application
+
+### Component Upgrade
+* [FISH-1504] - Upgrade MicroProfile JWT-Auth to 1.2.1
+* [FISH-786] - Integrate Security Connector with the Payara Platform
+
+### New Features
+* [FISH-1526] - Upgrade Payara Enterprise to MicroProfile 4.0
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc:24]: Should get changed to 4.1 - not 4.0

File: docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+* [FISH-1372] - MicroProfile Health 3.1
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc:25]: Not sure this is worth mentioning? It's a part of MicroProfile 4.1 - it's not a standalone feature.

**Review by Pandrex247 — CHANGES_REQUESTED** [review]
I feel like there should be some words at the top of the release notes highlighting that the upgrade from MicroProfile 3.3 to 4.1 introduces breaking changes.

File: docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+ * Java EE 8 Applications
+ * Jakarta EE 8
+ * MicroProfile 4.1
+
+=== Notes
+==== MicroProfile 4.1
+The release of Payara Enterprise 5.30.0 introduces an update of MicroProfile from 3.3 to 4.1, bringing with it some breaking changes. Details of these changes and how to work around them can be found within the documentation of each spec. 
+
+=== New Features
+* [FISH-1526] - Upgrade Payara Enterprise to MicroProfile 4.1
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc:14]: Sorry, I wasn't clear - I meant that this should get replaced with the MP 4.1 issue, not just changing the number.
```suggestion
* [FISH-666] - Upgrade Payara Enterprise to MicroProfile 4.1
```

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

**Review by Pandrex247 — APPROVED** [review]

**Review by debbiehoffman — APPROVED** [review]

**Review by rdebusscher — APPROVED** [review]

**Pull request merged**